serial_asyncio/__init__.py

```diff
@@ -75,9 +75,21 @@ def loop(self):
 
     @property
     def serial(self):
-        """The underlying Serial instance."""
+        """The underlying Serial instance.
+
+        Equivalent to get_extra_info("serial")
+        """
         return self._serial
 
+    def get_extra_info(name, default=None):
+        """Get optional transport information.
+
+        Currently only "serial" is available.
+        """
+        if name == "serial":
+            return self._serial
```

Comment:
uses `self` but that is missing from the methods parameter list... and this is not a `staticmethod`